src/code/modules/tutorials.js

```diff
@@ -1,6 +1,6 @@
 import Immutable from 'seamless-immutable';
 import actionTypes from '../action-types';
-import t from '../utilities/translate';
+// import t from '../utilities/translate';
```

Comment:
Commented out so can probably remove?

Reply:
In general I sometimes left code commented out if I thought that it might be a useful reminder to someone else, e.g. a potentially useful piece of code that isn't currently being called but might potentially be useful at some point. In this case, however, that argument doesn't seem to apply, so I'll remove it.